src/navs/index.js

```diff
@@ -1057,27 +1073,29 @@ export const handbookSidebar = [
                 url: '/handbook/growth/marketing',
             },
             {
-                name: 'Marketing ownership',
+                name: 'Who can help me?',
                 url: '/handbook/growth/marketing/ownership',
             },
             {
-                name: 'How we do video at PostHog',
-                url: '/handbook/growth/marketing/video',
-            },
-            {
-                name: 'Sponsorships',
+                name: 'Other',
```

Comment:
I don't see why we need to hide these, since there is so little content under marketing, we can probably just have them all listed and remove the "other" folder.